app/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "airbnb-base",
```

Comment:
I think this should be just "extends": "airbnb"

Reply:
I looked at the difference between https://www.npmjs.com/package/eslint-config-airbnb and https://www.npmjs.com/package/eslint-config-airbnb-base and found out that `airbnb` contains useful tools for React projects, while `airbnb-base` is recommended for projects that don't use React. See this comment:
![image](https://user-images.githubusercontent.com/108130/32190489-4870c70c-bdae-11e7-9959-181237f69c42.png)

Based on that, is then what I have correct @justafish?

Reply:
oh cool, I didn't know that! Thanks @juampynr ! 💃